sensor_ip: "192.168.12.12"
host_ip: "192.168.12.8"
host_udp_port: 0
frame_id: rear_lidar_link
skip: 0
angle_start: -2.3561944901923448
angle_end: 2.3561944901923448
time_offset: -0.065
channel_enabled: True
general_system_state: True
derived_settings: True
measurement_data: True
intrusion_data: True
application_io_data: True
use_persistent_config: False
udp_connection_monitor : True
udp_connection_monitor_watchdog_timeout_ms : 5000